src/icalendar/version.py

```diff
@@ -1,4 +1,28 @@
-"""Version file as a stable interface for the generated _version.py file."""
+"""Version information on icalendar.
+
+This module provides a stable interface to the generated _version.py file.
+
+Version as a string:
+
+.. code-block:: pycon
+
```

Comment:
Remove empty code-block and use markup.

```suggestion
This module provides a stable interface to the generated :file:`_version.py` file.

```